less/_grid.less

```diff
@@ -76,9 +76,13 @@
 
 // other CSS for the container of the new columns
 .grid {
+	@grid-sides: 15px;
 	.clearfix;
 
-	width: 100%;
+	padding: 0 25px 40px 25px;
```

Comment:
That's an interesting point. I get the impression that this class is supposed to represent just rows, so maybe we should make the class ```grid-row```?